src/test/java/com/redhat/lightblue/camel/utils/LightblueInsertRequestTransformer.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.lightblue.camel.transformer;
+package com.redhat.lightblue.camel.utils;
```

Comment:
We could continue using save in all cases once https://github.com/lightblue-platform/lightblue-core/issues/184 is implemented.